### ISSUE TITLE
review comorbs for characterisation table 1 and if others are wanted for study 

### ISSUE THREAD

**Issue by daniellenewby** [oxford-pharmacoepi/THIN_incidence_prevalence_colorectal_cancer#11]
As with this issue but for comorbs https://github.com/oxford-pharmacoepi/THIN_incidence_prevalence_colorectal_cancer/issues/8

**Comment by daniprietoalhambra** [comment]
Where are the jsons for comorbs please? I could find them easily in your medications issue. Thanks

**Comment by daniellenewby** [comment]
https://github.com/oxford-pharmacoepi/THIN_incidence_prevalence_colorectal_cancer/tree/main/2_Study/1_InstantiateCohorts/Conditions

Some of them i created when i was looking at all the different cancers so alot of them might not be relevant for CRC also not sure if patri added some more too :)

**Comment by daniprietoalhambra** [comment]
There are lots of irrelevant ones indeed, mostly precancerous lesions of other organs probably irrelevant for crc eg Barrett's oesophagus. How do you want me to edit? Should I just list the ones to delete here and you do that?

**Comment by daniellenewby** [comment]
yes please or whatever is easiest for you ie excel sheet then share here etc

**Comment by daniellenewby** [comment]
thank you :)

**Comment by daniellenewby** [comment]
@daniprietoalhambra if you have some time could you let me know which ones i can cut - thanks!

**Comment by daniprietoalhambra** [comment]
I would keep:

1. abdominal_bloating.json
2. abdominal_pain.json
3. anemia.json
7. barretts_esophagus.json
9. blood_in_urine.json
10. breast_lump.json
11. bronchitis.json
12. cancerexcludnonmelaskincancer.json
13. carcinoma_in_situ_breast.json
14. cerebrovascular_disease.json
16. chronic_kidney_disease.json
17. chronic_liver_disease.json
18. constipation.json
19. copd.json
21. cough.json
25. diarrhea.json
26. difficulty_breathing.json
27. dysphagia.json
30. family_history_any_malignancy.json
31. family_history_breast_cancer.json
32. family_history_colorectal_cancer.json
33. family_history_gi_malignancy.json
34. family_history_polyps.json
35. gallstones.json
36. gerd.json
37. gi_hemorrhage.json
40. heart_failure.json
41. heartburn.json
43. helicobacter_pylori_infection.json
44. hemochromatosis.json
45. hemoptysis.json
47. hereditary_crc_predisposing_sd.json
48. hiv.json
49. hoarse.json
50. hpv_infection.json
54. indigestion.json
55. inflammatory_bowel_disease.json
56. ischemic_heart_disease.json
57. jaundice.json
58. kidney_stones.json
59. liver_cirrhosis.json
61. menorrhagia.json
64. nafld.json
65. nausea.json
67. obesity_diagnosis.json
70. pain_in_breast.json
71. pain_in_pelvis.json
72. painful_sexual_intercourse.json
73. pancreatitis.json
74. pcos.json
75. peripheral_vas_disease.json
76. personal_history_polyps.json
77. pneumonia.json
78. polyp_large_intestine.json
79. psoriasis.json
80. pulmonary_embolism.json
81. rectal_bleeding.json
82. renal_impairment.json
83. rheumatoid_arthritis.json
84. sexually_transmitted_infections.json
85. sinusitis.json
86. sore_throat.json
87. stroke.json
88. swollen_abdomen.json
91. t_2_d.json
93. urinary_incontinence.json
94. uti.json
95. venous_thromboembolism.json
96. viral_hepatitis.json

Some will be relevant only for some specific cancers but I guess that's fine?

**Comment by daniellenewby** [comment]
Yes i was way more inclusive for the list of comorbs as the THIN application included lots of different cancers but if you want to cut it down to focus on CRC then that is ok too :)

**Comment by daniprietoalhambra** [comment]
OK. So for CRC I would only keep these:

- abdominal_bloating.json
- abdominal_pain.json
- anemia.json
- cancerexcludnonmelaskincancer.json
- cerebrovascular_disease.json
- chronic_kidney_disease.json
- chronic_liver_disease.json
- constipation.json
- copd.json
- diarrhea.json
- family_history_any_malignancy.json
- family_history_colorectal_cancer.json
- family_history_gi_malignancy.json
- family_history_polyps.json
- gallstones.json
- gi_hemorrhage.json
- heart_failure.json
- hereditary_crc_predisposing_sd.json
- hiv.json
- indigestion.json
- inflammatory_bowel_disease.json
- ischemic_heart_disease.json
- nausea.json
- obesity_diagnosis.json
- pcos.json
- peripheral_vas_disease.json
- personal_history_polyps.json
- polyp_large_intestine.json
- psoriasis.json
- pulmonary_embolism.json
- rectal_bleeding.json
- rheumatoid_arthritis.json
- swollen_abdomen.json
- t_2_d.json
- venous_thromboembolism.json

**Comment by daniellenewby** [comment]
Perfect! :) thank you so much!
